src/style.css

```
@@ -0,0 +1,106 @@
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+}
+
+.product-cards {
+  width: 200px;
+  height: 403px;
+  display: flex;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  box-shadow: 0 2px 5px rgba(0, 0, 0, 0.07);
+  border-radius: 5px;
+  flex-direction: column;
+  align-items: center;
+  position: absolute;
+  left: 45%;
+  font-family: Roboto ;
+  font-style: normal;
+  padding: 30px 14px 15px;
+}
+
+.product-img {
+  height: 134px;
+  width: 160px;
+}
+
+.product-title {
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+  margin: 45px 0 5px;
+  line-height: 15px;
+  color: #000;
+}
+
+.product-code {
+  font-weight: normal;
+  font-style: normal;
+  font-size: 11px;
+  align-self: flex-start;
+  /* margin-top: 1px; */
+  line-height: 13px;
+}
+
+.product-rating {
+  margin-top: 16px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+}
+
+.product-review,
+.v-price {
+  font-size: 11px;
+}
+
+.product-star {
+  display: flex;
+  margin-top: 1px;
+  margin-left: 1px;
```

Don't repeat margin property.
```suggestion
  margin: top right bottom left;
```